app/views/login.scala.html

```diff
@@ -153,12 +153,11 @@ <h2 id="error-summary-heading">@Messages("label.checkTheForm")</h2>
                         '_type -> "hidden"
                     )
 
-                @defining(DateTime.now.toString) { now =>
-                    <input type="hidden" name="start-time" value="@now" />
-                }
+                    @defining(DateTime.now.toString) { now =>
```

Comment:
In which timezone and format we are _printing_ time?

Reply:
I haven't changed that piece of code. All I did was format it as it was not lining up properly.